waffle-doppelganger/src/Doppelganger.json

```diff
@@ -0,0 +1,65 @@
+{
```

Comment:
We should not put json files to repository and build them instead as a part of build and/or test process, see [buildTestContracts.ts](https://github.com/EthWorks/Waffle/blob/fc645edd76024593ca97a80f551a6a595bea311d/waffle-cli/buildTestContracts.ts) for inspiration

(we should have a separate build and test process here)